.gitignore

```diff
@@ -1,10 +1,10 @@
+__pycache__/
+.DS_Store
 .idea/Figma2Py.iml
 .idea/inspectionProfiles/profiles_settings.xml
 .idea/modules.xml
 .idea/vcs.xml
-__pycache__/window.cpython-39.pyc
-.DS_Store
 .idea/misc.xml
 .idea/workspace.xml
-__pycache__/backend.cpython-39.pyc
+.vscode
 env
```

Comment:
There were a couple duplicate lines (`__pycache__/`), and I sorted the base folders alphabetically out of compulsory habit. I also ignored `.vscode` for good measure.

Reply:
Should I check the working further or it's okay ?

Reply:
My testing just consisted of changing the name of my Rectangle from `TextBox` to `TextArea`, then re-converting the Figma file. The output worked fine for me, but I'm not entirely sure if there are any differences functionally between `Entry` and `Text`.

I would recommend some more testing before merging, but this would be a start towards implementing it. Maybe a temporary branch would be in order?